plugins/source/slack/resources/services/conversations/conversation_histories_mock_test.go

```diff
@@ -0,0 +1,27 @@
+package conversations
+
+import (
+	"testing"
+
+	"github.com/cloudquery/cloudquery/plugins/source/slack/client/mocks"
+	"github.com/cloudquery/plugin-sdk/faker"
+	"github.com/golang/mock/gomock"
+	"github.com/slack-go/slack"
+)
+
+func buildConversationHistoriesMock(t *testing.T, m *mocks.MockSlackClient) {
+	d := &slack.GetConversationHistoryResponse{}
+	err := faker.FakeObject(&d)
+	if err != nil {
+		t.Fatal(err)
+	}
+	d.HasMore = false
+	msgs := make([]slack.Message, 1)
```

Comment:
no need to manually create the slice of len 1 (faker does it for you?), just do `var []slack.Message`

Reply:
I prefer being explicit about what the test is doing and the fact that we want exactly one message